docs/csharp/language-reference/compiler-messages/cs0019.md

```diff
@@ -18,7 +18,7 @@ Operator 'operator' cannot be applied to operands of type 'type' and 'type'
 
 ## Example
 
- In the following example, CS0019 is generated in two places because [bool](../builtin-types/bool.md) in C# is not convertible to [int](../builtin-types/integral-numeric-types.md). CS0019 also is generated when the subtraction operator `-` is applied to a string. The addition operator `+` can be used with string operands because that operator is overloaded by the `String` class to perform string concatenation.
+ In the following example, CS0019 is generated in three places because [bool](../builtin-types/bool.md) in C# is not convertible to [int](../builtin-types/integral-numeric-types.md). CS0019 also is generated when the subtraction operator `-` is applied to a string. The addition operator `+` can be used with string operands because that operator is overloaded by the `String` class to perform string concatenation.
```

Comment:
```suggestion
In the following example, CS0019 is generated in three places because [bool](../builtin-types/bool.md) in C# is not convertible to [int](../builtin-types/integral-numeric-types.md). CS0019 is also generated when the subtraction operator `-` is applied to a string. The addition operator `+` can be used with string operands because that operator is overloaded by the `String` class to perform string concatenation.
```